### ISSUE TITLE
NTR: Xanthine oxidase low (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8199]
New term request Xanthine oxidase low (segal_071024203507):   UMLS: 4315246

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000218
- label: Reduced circulating xanthine oxidase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:9153281
- def: Activity of the enzyme xanthine oxidase below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8199
- synonyms: Xanthine oxidase low(NONE)
